uploader/templates/uploader/component/places.html

```diff
@@ -0,0 +1,48 @@
+
+<div id="people">
+    <h1>People and groups</h1>
+
+    <p>Details of {{ people| length }} new {{people|pluralize:"person, people"}} or group{{people|pluralize:",s"}}
+        {{people|pluralize:"was, were"}} uploaded in this contribution. </p>
+</div>
```

Comment:
places.html but people content, will that be implement later?

Reply:
yes, I'll implement later